content/blog/2018/02/2018-02-22-cheetah.adoc

```diff
@@ -0,0 +1,134 @@
+---
+:layout: post
+:title: "Project Cheetah - Faster, Leaner Pipeline That Can Keep Up With Demand"
```

Comment:
That's a fair concern -- much like B.O. and Remoting, I'm trying to surface this as a public-facing project rather than a purely-internal codename.  This is because it has its multiple parts and a longer-running scope involving work from multiple people -- so it's easy to refer to by project name than "Durability / Scalability / Pipeline Storage and Logging + ClassLoading and other goodies."

What that means is that I'm going to have to go through and tag JIRAs, etc appropriately.  Currently this tracks against JIRA epic [JENKINS-47170](https://issues.jenkins-ci.org/browse/JENKINS-47170)